guides/Releasing Expo Go for Android.md

```diff
@@ -31,12 +31,12 @@ This document will guide you through the process of releasing a new version of E
     **Why:** So that developers who used `expo-cli` to download Expo Go to their devices can download the update.
 
     **How:**
-    - Open CircleCI on the release branch and go to the `client` workflow. Once `client_android` job is finished, approve `client_android_apk_release_approve` job and follow the next job `client_android_apk_release` which takes and uploads the artifact archive from `client_android` job to staging.
+    - Open GitHub Checks for the release branch and go to the `Android Client / build` workflow. Once it's finished, run `et wd client-android-simulator` and follow that workflow run which takes and uploads the client APK to staging.
```

Comment:
Since we use GitHub Actions, `client-android-simulator` also runs the actual build (not just uploads the artifact from the job before approval as in CircleCI), so now there is no need to track GitHub checks before dispatching the workflow.

```suggestion
    - Run `et workflow-dispatch client-android-simulator` and follow that workflow run which builds Expo Go for simulator and uploads the APK to staging.
```